docs/install/2-install-server-with-docker.rst

```diff
@@ -291,9 +275,16 @@ There are two ways you can achieve this:
             image: ghcr.io/inmanta/orchestrator:latest
             ports:
                 - 8888:8888
+            env_file: ./resources/my-env-file
+            environment:
+                INMANTA_DATABASE_HOST: inmanta_db
```

Comment:
I did add both ways to do it in the example, they can actually be combined and this clearly show how to use both of them. Not sure if this is alright.